src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -144,7 +144,7 @@ const ArtworkAuctionCreateAlertHeader: FC<ArtworkAuctionCreateAlertHeaderProps>
                 Icon={BellStrokeIcon}
                 // @ts-ignore
                 as={RouterLink}
-                to={"/settings/alerts"}
+                to={"/favorites/alerts"}
```

Comment:
this makes sense to me more than the `settings/alerts` - I would only suggest ensuring we align that with Eigen here and marketing to not break any deep links logic. cc @dblandin 

https://github.com/artsy/eigen/blob/9c5798f08c98d86739d7477006b9cdea2beb044e/src/app/routes.ts#L254

Reply:
There would be redirects so it shouldn't be an issue.